Need a test for wget if it is required for testing.

Because of https://github.com/kubernetes-incubator/service-catalog/pull/1384 `make test` now fails on my Mac.

```
<snip>
Waiting for API Server to be available...
Timed-out waiting for API Server
+ wget --ca-certificate ~/go/src/github.com/kubernetes-incubator/service-catalog/.var/run/kubernetes-service-catalog/apiserver.crt https://localhost:32784
~/go/src/github.com/kubernetes-incubator/service-catalog/contrib/hack/start-server.sh: line 60: wget: command not found
Cleaning up
make: *** [test-integration] Error 127
```

--- a/contrib/hack/start-server.sh
+++ b/contrib/hack/start-server.sh
@@ -52,12 +52,12 @@ count=0
 D_HOST=${DOCKER_HOST:-localhost}
 D_HOST=${D_HOST#*//}   # remove leading proto://
 D_HOST=${D_HOST%:*}    # remove trailing port #
-while ! wget --ca-certificate ${ROOT}/.var/run/kubernetes-service-catalog/apiserver.crt https://${D_HOST}:${PORT} > /dev/null 2>&1 ; do
+while ! curl --cacert ${ROOT}/.var/run/kubernetes-service-catalog/apiserver.crt https://${D_HOST}:${PORT} > /dev/null 2>&1 ; do
 	sleep 1
 	(( count++ )) || true
 	if [ "${count}" == "30" ]; then
 		echo "Timed-out waiting for API Server"
-		(set -x ; wget --ca-certificate ${ROOT}/.var/run/kubernetes-service-catalog/apiserver.crt https://${D_HOST}:${PORT})
+		(set -x ; curl --cacert ${ROOT}/.var/run/kubernetes-service-catalog/apiserver.crt https://${D_HOST}:${PORT})
 		(set -x ; docker ps)
 		(set -x ; docker logs apiserver)
 		exit 1

